.github/workflows/build-hw-reusable.yml

```diff
@@ -29,14 +29,18 @@ jobs:
     strategy:
       matrix:
         adapter: [
-          {name: "${{inputs.name}}", platform: "${{inputs.platform}}"},
+          {name: "${{inputs.name}}", platform: "${{inputs.platform}}", static: OFF},
         ]
         build_type: [Debug, Release]
         compiler: [{c: gcc, cxx: g++}, {c: clang, cxx: clang++}]
         # TODO: The latest L0 loader segfaults when built with clang.
         exclude:
          - adapter: {name: L0, platform: ""}
            compiler: {c: clang, cxx: clang++}
+        include:
```

Comment:
It seems that this adds L0 HW job to `OpenCL, HIP, CUDA ...` runners.

Reply:
done.